torch/__init__.py

```diff
@@ -309,6 +309,10 @@ def compiled_with_cxx11_abi():
     r"""Returns whether PyTorch was built with _GLIBCXX_USE_CXX11_ABI=1"""
     return _C._GLIBCXX_USE_CXX11_ABI
 
+def parallel_info():
```

Comment:
the only style question I have is `torch.parallel_info()` vs `torch.__config__.parallel_info()`. @gchanan  - thoughts?

Reply:
I'd vote for not adding it in the torch namespace,  but instead in `__config__`.

Reply:
I'd put it in `__config__`.